docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-linux.md

```diff
@@ -10,9 +10,98 @@ If you do not have a DirectX 11 capable GPU in your system (for example if you a
+
+Easiest way to do this is to copy the entire Titanfall2 folder to the server and delete the [following files](https://github.com/pg9182/northstar-dedicated#reducing-the-size-), or alternatively download and run [this shell script](https://gist.github.com/pg9182/9a962adbfc27e93237cd14e4523c9da8) to download only the files necessary to run a server.
+
+#### Copy Titanfall Folder over to the Linux Machine
+
+1. Access the files on your Linux machine using tools like [WinSCP](https://winscp.net/eng/download.php) or any tool that allows for SSH file transfer, or transfer files via USB drive if you can physically access your server.
+2. Navigate to the folder where you want to store the files. You can put them into `~/Titanfall2` for example.
+3. Copy the newly pruned Titanfall folder to the server.
+
+![Game files copied to `/mnt/Titanfall` as an example](https://i.postimg.cc/15HbbzFr/image.png)
+
+#### Copy mods to server
+
+If you have configured some mods,these can be placed at a similar location, like `~/mods` for example
+![Mods placed inside /mnt/mods](https://i.postimg.cc/tRD5jnrJ/image.png)
+
+#### Create docker-compose file
+
+We are going to be using Docker-Compose to set up our container. This gives us much more flexibility and allows us to make changes to the start up arguments much cleaner. This example will let you start your servers without needing to have your compose in the same folder and directory as Titanfall 2 and your mods folder. Create a compose file in your home directory, or wherever is most convenient.
+
+```
+nano docker-compose.yml
+```
+
+Example `docker-compose.yml` \
+(Note that you will have to adjust some lines to make it work on your machine)
+
+```yaml
+version: '3'
+services:
+ northstar-attrition: 
+   image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0 
+   pull_policy: always 
+   environment:
+     - NS_PORT=37015
+     - NS_PORT_AUTH=8081
+     - 'NS_SERVER_NAME=Enter Server Name here'
+     - 'NS_SERVER_DESC=Enter your description here'
+     - NS_EXTRA_ARGUMENTS=
+       +setplaylist aitdm # Attrition
+       +mp_gamemode aitdm # Attrition
+       +map mp_angel_city
+       +ns_private_match_countdown_length 0
+       +ns_should_return_to_lobby 0
+       +net_compresspackets_minsize 64
+       +net_compresspackets 1
+       +spewlog_enable 0
+       +sv_maxrate 127000
+   volumes:
+     - /home/YOUR_USERNAME_HERE/Titanfall2:/mnt/titanfall:ro
+     - /home/YOUR_USERNAME_HERE/Titanfall2/mods:/mnt/mods:ro
+   ports:
+     - '37015:37015/udp'
+     - '8081:8081/tcp'
+   restart: always
+```
+
+A list of all the CONVARs are [here](../../hosting-a-server-with-northstar/dedicated-server#convars)
+
+### Starting
+
+#### Run the following command
+
+To run this container go to the folder you saved the `docker-compose.yml` in and type
+
+```
+docker-compose up
+```
+
```

Comment:
Maybe a final line of, "your server should be up and running now. Check the in-game server browser or online server list \<link here\> to see if it's online. Should your server not show up or should you encounter any other issues, check the server troubleshooting section \<link here\>." or something like that to round of the section.

Reply:
Also maybe a mention that pg's README contains more advanced examples like running multiple servers using a single set of Titanfall2 game files, all in a single compose file.

Reply:
added both an ending statement linking to the server browser & a link to pg's complex docker-compose